dev/app/init-builtin-widgets.js

```diff
@@ -296,25 +327,34 @@ export default () => {
       );
     }));
 
-  storiesOf('Menu').add('with show more', wrapWithHits(container => {
-    window.search.addWidget(
-      instantsearch.widgets.menu({
-        container,
-        attributeName: 'categories',
-        limit: 3,
-        showMore: {
+  storiesOf('Menu')
+    .add('Default', wrapWithHits(container => {
+      window.search.addWidget(
+        instantsearch.widgets.menu({
+          container,
+          attributeName: 'categories',
+        })
+      );
+    }))
+    .add('with show more and header', wrapWithHits(container => {
+      window.search.addWidget(
+        instantsearch.widgets.menu({
+          container,
+          attributeName: 'categories',
+          limit: 3,
+          showMore: {
```

Comment:
Not really, it's simple text like ris see: https://community.algolia.com/react-instantsearch/storybook/?selectedKind=HierarchicalMenu&selectedStory=with%20show%20more&full=0&down=1&left=1&panelRight=1&downPanel=storybooks%2Fstorybook-addon-knobs